### PR TITLE
dep: update vendor

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -31,84 +28,57 @@ var (
...
-	{
-		loggerConfig := micrologger.DefaultConfig()
-		loggerConfig.IOWriter = os.Stdout
-		newLogger, err = micrologger.New(loggerConfig)
-		if err != nil {
-			return microerror.Maskf(err, "micrologger.New")
-		}
-	}
+	ctx := context.Background()
+	logger, err := micrologger.New(micrologger.Config{})
```
Comment (xh3b4sd) [inline main.go]: There is now error handling missing.

File: main.go [hunk 2]
```diff
@@ -118,19 +88,19 @@ func mainWithError() error {
...
-		commandConfig.Name = name
-		commandConfig.Source = source
+		c := command.Config{
+			Logger:        logger,
+			ServerFactory: serverFactory,
+
+			Description: description,
+			GitCommit:   gitCommit,
+			Name:        name,
+			Source:      source,
```
Comment (xh3b4sd) [inline main.go]: We can also provide version bundles to make them available for the version command of the daemon.

File: server/server.go [hunk 3]
```diff
@@ -10,83 +10,81 @@ import (
...
+		logger: config.Logger,
+
+		bootOnce: sync.Once{},
+		config: microserver.Config{
+			Logger:      config.Logger,
+			ServiceName: config.ProjectName,
+			Viper:       config.Viper,
+
+			Endpoints: []microserver.Endpoint{
+				endpointCollection.Version,
```
Comment (xh3b4sd) [inline server/server.go]: Based on the diff the health endpoint if missing.

**Comment by kopiczko** [comment]
Thanks for the comments. I addressed all of that in separate PRs
https://github.com/giantswarm/azure-operator/pull/152
https://github.com/giantswarm/azure-operator/pull/153
https://github.com/giantswarm/azure-operator/pull/154

**Pull request merged**